### PR TITLE
clarify that questions should be good questions

### PR DESCRIPTION
**Pull request by greggles**
Trying to briefly state some of the elements in this portion of the talk https://youtu.be/PJjmw9TRB7s?t=944 - could probably be improved.

### CODE REVIEW

File: code-review/README.md [hunk 1]
```diff
@@ -8,8 +8,9 @@ Everyone
 
 * Accept that many programming decisions are opinions. Discuss tradeoffs, which
   you prefer, and reach a resolution quickly.
-* Ask questions; don't make demands. ("What do you think about naming this
+* Ask good questions; don't make demands. ("What do you think about naming this
   `:user_id`?")
+* Good questions avoid judgment and avoid assumptions about the author's perspective
```
Comment (gylaz) [inline code-review/README.md]: Should there be a period at the end?

Comment (mike-burns) [inline code-review/README.md]: Is it possible to include an example, like some of the others have?

**Comment by mike-burns** [comment]
Maybe we should link to that video. Do we link to it elsewhere?

File: code-review/README.md (hunk 1 above)
Comment (greggles) [inline code-review/README.md]: @gylaz good call, added.

@mike-burns not sure, I think the previous line has an example of a good question so it doesn't feel as important to me to have one on this line.

**Comment by greggles** [comment]
It makes sense to me to add a link to the video in the first paragraph section. Maybe:

"Watch a presentation that covers this material from [Derek Prior at RailsConf 2015](https://www.youtube.com/watch?v=PJjmw9TRB7s)."

**Comment by jamescmartinez** [comment]
Should we merge this in?

**Comment by croaky** [comment]
Merged as 2ca8be6

**Pull request closed without merging**